crm-app/src/store/store.js

```diff
@@ -0,0 +1,34 @@
+import {createStore, combineReducers, applyMiddleware} from 'redux';
```

Comment:
{createStore, combineReducers, applyMiddleware} -> { createStore, combineReducers, applyMiddleware }

Reply:
I used to write an empty space after '{', but you told me the Enact code convention is w/o space when starting with an open curly brace. I follow the rule. I think I fixed them...hopefully..